site/content/docs/4.3/examples/masonry/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: single
+title: Bootstrap grid masonry example
+description: This example illustrates how to integrate the [Masonry plugin](https://masonry.desandro.com/) with the Bootstrap grid. More options & documentation can be found on their [documentation site](https://masonry.desandro.com/).
+include_docs_stylesheets: true
+active_menu: "example"
+extra_js:
+  - src: "https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js"
+    integrity: "sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI="
+    async: true
+---
+
+Masonry is not included in Bootstrap, but can easily be added by including your locally hosted file or using the following hosted javascript file:
+```js
+<script src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous" async></script>
```

Comment:
I realize we're using CloudFlare CDN for every extra JS which specifies an exact version right now. Thinking about maintainance effort:
with an unspecific version naming, we don't have to update the docs everytime a new version is out, see https://masonry.desandro.com/ CDN example

Reply:
We do the same anyway in another example, with hardcoded versions, so I'd say it's fine for now.

Ideally, maybe we should move all this in a data file or something so that it's not so deeply hidden.